src/index.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
do you plan to write the utils in typescript so we can generate the declarations and include them in the NPM package?

Reply:
Yes. But that is part of a much larger epic.